refine readme, update updates (#1648)
infiniflow/ragflow: README.md

## 🔥 Latest Updates

> - 2024-07-23 Support parsing audio files.

Reviewer: ```suggestion
- 2024-07-23 Supports audio file parsing.

```

> - 2024-07-21 Support more LLM suppliers(LocalAI/OpenRouter/StepFun/Nvidia).

Reviewer: ```suggestion
- 2024-07-21 Supports more LLMs (LocalAI, OpenRouter, StepFun, and Nvidia).

```
Nvidia is not an LLM.

> - 2024-07-18 Add components(Wikipedia/PubMed/Baidu/Duckduckgo) to the graph.

Reviewer: ```suggestion
- 2024-07-18 Adds more components (Wikipedia, PubMed, Baidu, and Duckduckgo) to the graph.

```
